thirdeye/thirdeye-frontend/app/mocks/anomalyFunction.js

```diff
@@ -0,0 +1,72 @@
+export default [
+  {
+    "id": 10,
+    "version": 1,
+    "createdBy": null,
+    "updatedBy": null,
+    "collection": "test_collection_1",
+    "functionName": "test_function_1",
+    "metric": "test_metric_1",
+    "metrics": ["test_metric_1"],
+    "metricFunction": "SUM",
+    "type": "WEEK_OVER_WEEK_RULE",
+    "isActive": true,
+    "globalMetric": null,
+    "globalMetricFilters": null,
+    "properties": "test_properties_1",
+    "cron": "test_cron_1",
+    "frequency": {
+      "size": 1,
+      "unit": "DAYS"
+    },
+    "bucketSize": 1,
+    "bucketUnit": "DAYS",
+    "windowSize": 7,
+    "windowUnit": "DAYS",
+    "windowDelay": 4,
+    "windowDelayUnit": "HOURS",
+    "exploreDimensions": null,
+    "filters": null,
+    "metricId": 0,
+    "dataFilter": null,
+    "alertFilter": null,
+    "anomalyMergeConfig": null,
+    "requiresCompletenessCheck": true,
+    "topicMetric": "test_metric",
+    "toCalculateGlobalMetric": false
+  }, {
+    "id": 20,
+    "version": 1,
+    "createdBy": null,
+    "updatedBy": null,
+    "collection": "test_collection_2",
+    "functionName": "test_function2",
+    "metric": "test_metric_2",
+    "metrics": ["test_metric_2"],
+    "metricFunction": "SUM",
+    "type": "WEEK_OVER_WEEK_RULE",
+    "isActive": true,
+    "globalMetric": null,
+    "globalMetricFilters": null,
+    "properties": "test_properties_2",
+    "cron": "test_cron_2",
+    "frequency": {
+      "size": 1,
+      "unit": "DAYS"
+    },
+    "bucketSize": 1,
+    "bucketUnit": "DAYS",
+    "windowSize": 7,
+    "windowUnit": "DAYS",
+    "windowDelay": 4,
+    "windowDelayUnit": "HOURS",
+    "exploreDimensions": "platform",
+    "filters": "platform=desktop;platform=tablet",
```

Comment:
nitpick: let's obfuscate this as well.

Reply:
By obfuscate, do you mean "delete" or change the data to something more obscure?

Reply:
changing the data to something else `"dress=froufrou; hat=fancy"`